server/lib/actors/fetcher.go

```diff
@@ -0,0 +1,81 @@
+package actors
+
+import (
+	"fmt"
+	"database/sql"
+	"time"
+
+	"github.com/umegaya/channer/server/lib/models"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/channer/server/lib/models (can't find import: github.com/umegaya/channer/server/lib/models)